upp-elasticsearch-provisioner/vars/upp-ccf-content-os-prod-us.yml

```diff
@@ -3,6 +3,6 @@ template_params:
   TeamDLTag: "universal.publishing.platform@ft.com"
   SystemCodeTag: "upp"
   ClusterName: "{{cluster_name}}"
-  InstanceType: "m6g.large.search"
+  InstanceType: "m6g.xlarge.search"
   MasterType: "m6g.large.search"
   StorageSize: 30
```

Comment:
`question`: There is no newline at the end of the file, will this affect the provisioning?